lib/components/viewers/nearby/styled.tsx

```diff
@@ -108,3 +109,30 @@ export const Scrollable = styled.div`
   height: 100%;
   overflow-y: scroll;
 `
+
+export const CardFooter = styled.div`
+  background: rgba(0, 0, 0, 0.2);
+  border-bottom-left-radius: 10px;
+  border-bottom-right-radius: 10px;
+  color: ${grey[700]};
+  font-size: 15px;
+  padding: 5px 12px;
+  margin-top: -6px;
+
+  span {
+    padding-top: 5px;
+    display: block;
+  }
+
+  &::before {
```

Comment:
Is this for the margin between the footer and the body of the card, and can it be done using `margin-top` of `CardFooter` itself instead?

Reply:
This is used to create the border radius effect on the body of the card, so it looks like the footer is peeking out from below it. 
![image](https://github.com/user-attachments/assets/9d626c0c-6f00-427f-b48b-ed407ada0409)